naptha_sdk/configs.py

```diff
@@ -31,37 +31,37 @@ async def load_module_config_data(deployment, load_persona_data=False, load_pers
 
     return deployment
 
-def load_subdeployments(deployment, node_url):
+async def load_subdeployments(deployment, node_url):
 
-    configs_path = Path(f"{Path.cwd().name}/configs")
```

Comment:
path cwd is required for running module repos locally (the part in `if __name__ == "__main__"`)